src/java/voldemort/VoldemortClientShell.java

```diff
@@ -151,21 +151,38 @@ public void process(boolean printCommands) {
     public static void main(String[] args) throws Exception {
 
         OptionParser parser = new OptionParser();
-        parser.accepts("client-zone-id", "client zone id for zone routing")
+        parser.accepts("client-zone-id", "Client zone id for zone routing")
               .withRequiredArg()
               .describedAs("zone-id")
               .ofType(Integer.class);
+        parser.accepts("config-file", "A properties file that contains client config properties")
+              .withRequiredArg()
```

Comment:
Does this mean the new config parameter is required (i.e.: mandatory)? Can we make it optional instead, so as not to break existing users who may not wish to override it?